proxy/provider_darwin.go

```diff
@@ -45,39 +45,74 @@ If none is found, or an error occurs, nil is returned.
 This function searches the following locations in the following order:
 	* Configuration file: proxy.config
 	* Environment: HTTPS_PROXY, https_proxy, ...
-	* scutil: The Network settings
 Params:
-	protocol: The proxy's protocol (i.e. https)
+	protocol: The protocol of traffic the proxy is to be used for. (i.e. http, https, ftp, socks)
 	targetUrl: The URL the proxy is to be used for. (i.e. https://test.endpoint.rapid7.com)
 Returns:
 	Proxy: A proxy was found
 	nil: A proxy was not found, or an error occurred
 */
 func (p *providerDarwin) Get(protocol string, targetUrlStr string) Proxy {
 	targetUrl := ParseTargetURL(targetUrlStr, protocol)
-	if proxy := p.provider.get(protocol, targetUrl); proxy != nil {
+	proxy := p.provider.get(protocol, targetUrl)
+	if proxy != nil {
```

Comment:
Why was this change made?

Reply:
Mistake during alignment of other platforms, updated